roxie/ccd/ccdsnmp.cpp

```diff
@@ -908,7 +908,6 @@ class CQueryStatsAggregator : public CInterface, implements IQueryStatsAggregato
     CQueryStatsAggregator(const char *_queryName, unsigned _expirySeconds)
         : queryName(_queryName)
     {
-        SpinBlock b(queryStatsCrit);
```

Comment:
It was removed in error.  I'll add it back in again.

I had removed it because in general there is no point having a critical block inside a constructor.  However I see this was protecting adding the aggregator to the array so it is needed.  I'll also add a comment to indicate why it is needed.